# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libnl/0001-include-byteswap.h.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Include byteswap.h to have __bswap_64()

--- a/lib/netfilter/log.c	2014-04-07 17:47:17.721864920 +0200
+++ b/lib/netfilter/log.c	2014-04-07 17:51:20.215496545 +0200
@@ -18,6 +18,7 @@
  * @{
  */
 
+#include <byteswap.h>
 #include <sys/types.h>
 #include <linux/netfilter/nfnetlink_log.h>
 
